File: HACKATHON/js/insurance.js
// Insurance Planning
function addPolicy(type, premium, coverage, renewalDate) {
  let policies = JSON.parse(localStorage.getItem("policies")) || [];
  policies.push({ type, premium, coverage, renewalDate });
  localStorage.setItem("policies", JSON.stringify(policies));
}

function showPolicies() {
  let list = JSON.parse(localStorage.getItem("policies")) || [];
  let output = "<h3>Insurance Policies</h3><ul>";
  list.forEach(p => {
    output += `<li>${p.type}: Premium ₹${p.premium}, Coverage ₹${p.coverage}, Renewal ${p.renewalDate}</li>`;
  });
  output += "</ul>";
  document.getElementById("content").innerHTML = output;
}

File: HACKATHON/java/Insurance.java
public class Insurance {
    private String type;   // Health, Life, Motor, Property
    private double premium;
    private double coverage;
    private String renewalDate;

    public Insurance(String type, double premium, double coverage, String renewalDate) {
        this.type = type;
        this.premium = premium;
        this.coverage = coverage;
        this.renewalDate = renewalDate;
    }

    public String getRenewalReminder() {
        return "Policy " + type + " renews on " + renewalDate;
    }

    public double getPremium() {
        return premium;
    }

    public double getCoverage() {
        return coverage;
    }

}
